src/containers/Programs/index.tsx

```diff
@@ -287,6 +310,7 @@ const mapStateToProps = (state: RootReducerInterface) => ({
   isLoadingPrograms: state.ProgramsReducer.programs.isLoadingPrograms,
   searchedInformation: state.ProgramsReducer.programs.searchedInformation,
   filteredInformation: state.ProgramsReducer.programs.filteredInformation,
+  favouriteInformation: state.ProgramsReducer.programs.filteredInformation,
```

Comment:
I am not sure we use favourites via reducers.

Reply:
will take this out